lookout/style/typos/utils.py

```diff
@@ -2,22 +2,19 @@
 
 from itertools import chain
 import lzma
-from typing import Dict, Iterable, List, Tuple
+from typing import Dict, Iterable, List, NamedTuple, Tuple
 
 from gensim.models.fasttext import FastText
 import numpy
 import pandas
 
-AFTER_COLUMN = "after"
-BEFORE_COLUMN = "before"
-CANDIDATE_COLUMN = "candidate"
-FEATURES_COLUMN = "features"
-CORRECT_TOKEN_COLUMN = "identifier"
-ID_COLUMN = "id"
-PROBABILITY_COLUMN = "proba"
-SPLIT_COLUMN = "token_split"
-SUGGESTIONS_COLUMN = "suggestions"
-TYPO_COLUMN = "typo"
+Columns = NamedTuple(
+    "Columns",
+    [("Token", str), ("CorrectToken", str), ("Split", str), ("CorrectSplit", str), ("After", str),
+     ("Before", str), ("Id", str), ("Candidate", str), ("Features", str), ("Probability", str),
+     ("Suggestions", str)])(
+    "token", "correct_token", "token_split", "correct_token_split", "after", "before", "id",
+    "candidate", "features", "proba", "suggestions")
```

Comment:
Why do you use `NamedTuple`?

I think it is better to use Enum in such case. 
```python
from enum import Enum, unique


@unique
class Columns(Enum):
    """
    DOC HERE
    """
    Token = "token"
    CorrectToken = "correct_token"
    ...
```

Reply:
Because this way I will have to write, for example, `Columns.Token.value` instead of `Columns.Token`. Looks like Enum with string fields can only work like that, I checked.